replace_refs.py

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+import linecache
+
+description="replaces the REF allele with the old REF allele if REF = ALT in the remapped VCF"
+
+parser = argparse.ArgumentParser(description = description, formatter_class=RawTextHelpFormatter)
+parser.add_argument("-i", "--vcf", help="VCF file containing remapped variants")
+parser.add_argument("-r", "--oldrefalleles", help="file containing the same header as the VCF file + the old REF \
+        alleles in the same order as the VCF")
```

Comment:
When splitting string literals, it's not necessary (and actually discouraged) to use the backslash sign. It's better to do it this way:

```python
some_call("some_string_literal_"
          "continuation_of_literal")
```

This will create `some_string_literal_continuation_of_literal`